interfaces/cython/cantera/thermo.pyx

```diff
@@ -900,17 +999,35 @@ cdef class ThermoPhase(_SolutionBase):
         :param basis:
             Determines if ``fuel`` and ``oxidizer`` are given in mole fractions
             (``basis='mole'``) or mass fractions (``basis='mass'``)
-        """
+        :param include_species:
+            List of species names. Only these species are considered for the
+            computation of the equivalence ratio. The default ``None`` means all
+            species are considered
```

Comment:
"List of species names (optional)." and 
"By default, all species are considered." avoids a technical detail of the implementation.